src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -917,6 +917,18 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                             $null = $output.Append($newline)
                                             $null = $output.Append($prop.Value)
                                         }
+                                        # Handle `BoundParameters` to show as Key/Value pairs, need to check the type name as the type is internal
+                                        elseif ($prop.Value.GetType().Name -eq 'PSBoundParametersDictionary') {
```

Comment:
Since the rest of the file already uses this pattern it's probably fine, but any idea why we didn't do `-is [System.Collections.IDictionary]` for both?

Reply:
I don't think there was any good reason for it.  No one caught it previously.  I'll generalize the code below it as it's essentially the same.  Good catch.